### PR TITLE
Fkwp/cleanup config

### PR DESCRIPTION
**Pull request by fkwp**

### CODE REVIEW

File: config/element_io_develop.json
Comment (toger5) [inline config/element_io_develop.json:1]: Is this not used anymore? Because this is maintained in elements config repo?

**Review by toger5 — APPROVED** [review]
Makes so much more sense (calling it netlify).

I have one question about the complete removal of develop.
If that is intentional and easy to answer feel free to merge.

**Comment by toger5** [comment]
I took the liberty to run prettier on the changes.

File: config/element_io_develop.json
Comment (fkwp) [reply config/element_io_develop.json:1]: its not used anymore since we use the docker image for our deployments

**Pull request merged**